packages/react/src/themes/types.ts

```diff
@@ -209,15 +209,18 @@ export interface ICSSPseudoElementStyle extends ICSSInJSStyle {
   content?: string
 }
 
-export interface StardustAnimationName {
-  keyframe?: any
-  params?: object
+type StardustAnimationKeyFrame = Record<string, React.CSSProperties>
```

Comment:
Aren't there typings for keyframes coming from CssInJS/fela?

Reply:
Fela exports `TKeyFrame`, but it's a type for a function 😢 Reworked these type to this one:

```jsx
type AnimationKeyFrame = Record<'from' | 'to' | string, ICSSInJSStyle>

export interface StardustAnimationName<P = Record<string, any>> {
  keyframe?: AnimationKeyFrame | ((params: P) => AnimationKeyFrame)
  params?: P
}
```